catalyst/dl/experiment/config.py

```diff
@@ -535,6 +542,14 @@ def get_callbacks(self, stage: str) -> "OrderedDict[Callback]":
             if not is_already_present:
                 callbacks[callback_name] = callback_fn()
 
+        # Remove LoggerCallback on worker nodes
+        if get_rank() > 0:
+            to_del = (LoggerCallback, ConfusionMatrixCallback)
```

Comment:
@Scitator can we mark all unpure callbacks like `ConfusionMatrixCallback` to not hardcode it here?

example
```
class ConfusionMatrixCallback(Callback, UnPure):
```

and here do
```
if get_rank() > 0:
            to_del = (UnPure)
```

Reply:
it's better no name them `NonDistributed`, or `DistributedMasterOnly`, 
looks like a good idea for future improvements